src/data/db.json

```diff
@@ -0,0 +1,9 @@
+{
+  "posts": [
+    { "id": 1, "title": "json-server", "author": "typicode" }
+  ],
+  "comments": [
+    { "id": 1, "body": "some comment", "postId": 1 }
+  ],
+  "profile": { "name": "typicode" }
+}
```

Comment:
new line at the end of file is required

Reply:
I think that we need move data directory to root of our project, because this is not part of our app, we need this only for testing

Reply:
ok, done.